app/models/manageiq/providers/redhat/infra_manager/ovirt_services.rb

```diff
@@ -846,9 +850,10 @@ def calculate_max_memory(vm, memory)
     #
     def add_vm_disks(vm_service, disk_specs)
       storage_spec = disk_specs[:storage]
+      default_disk_spec = disk_specs[:default] || {}
```

Comment:
As far as I know add_vm_disks is called only by vm_reconfigure for now, but it's ok to put the empty hash.